index.Rmd

```diff
@@ -76,8 +79,10 @@ if(!"patchwork" %in% installed.packages()){
   remotes::install_github("thomasp85/patchwork")
 }
 
-# To be included until new version with gg_parallel_slopes() is on CRAN
-remotes::install_github("moderndive/moderndive")
+if(!"moderndive" %in% installed.packages()){
+  # To be included until new version with gg_parallel_slopes() is on CRAN
+  remotes::install_github("moderndive/moderndive")
```

Comment:
Later on we should set `moderndive` to be the CRAN version only if possible, since most users will be installing that version.

Reply:
Yep. This was really just to get it so I didn’t need internet.

Reply:
I figured!